flexget/plugins/input/imdb_list.py

```diff
@@ -54,8 +57,9 @@ def on_task_input(self, task, config):
         else:
             url = 'http://www.imdb.com/list/%s' % config['list']
 
-        log.debug('Requesting: %s' % url)
-        page = task.requests.get(url, params=params)
+        headers = {'Accept-Language': config.get('force_language')}
```

Comment:
Maybe we can also find a bunch of tickets about imdb language to close if indeed this works. We should also probably see about adding it to imdb_lookup.